runtime/launcher.m

```diff
@@ -464,8 +464,14 @@
 			exe_path = strdup ([exePath UTF8String]);
 		 } else {
 
+		NSString * name= [[NSString stringWithUTF8String: data->basename] stringByAppendingString: @".dll"];
```

Comment:
I am uncertain if this is correct. I am uncertain why it started to be needed (but it works well enough to unbreak things for now).

Thoughts?